code/modules/organs/internal/eyes.dm

```diff
@@ -93,3 +93,7 @@
 
 /obj/item/organ/internal/eyes/proc/additional_flash_effects(var/intensity)
 	return -1
+
+/obj/item/organ/internal/eyes/emp_act(severity)
+	..()	//Returns if the organ isn't robotic
```

Comment:
No it doesn't.

Reply:
There literally is no return statement here.   The **parent** will return if its not robotic, but the parent returning does not cause this proc to end.  The following line applying eye_blurry will execute whether the organ is robotic or not.

Reply:
Ah, I see where I goofed. Fix will be up soonish.